eclipse-collections-code-generator/src/main/resources/api/set/mutablePrimitiveSet.stg

```diff
@@ -72,6 +72,16 @@ public interface Mutable<name>Set extends Mutable<name>Collection, <name>Set
     {
         throw new UnsupportedOperationException("Implement in concrete classes.");
     }
+
+    /**
+     * @since 11.0.
+     */
+    @Override
+    default Mutable<name>Set union(<name>Set set)
+    {
+        Mutable<name>Set newSet = this.toSet();
```

Comment:
use the larger of the two sets as starting point.

Reply:
Updated. Thanks for the suggestion!